test/extensions/bootstrap/wasm/test_data/speed_cpp.cc

```diff
@@ -15,6 +17,123 @@ START_WASM_PLUGIN(WasmSpeedCpp)
 
 int xDoNotRemove = 0;
 
+google::protobuf::Arena arena;
+
+google::protobuf::Struct args;
+google::protobuf::Struct* args_arena =
+    google::protobuf::Arena::CreateMessage<google::protobuf::Struct>(&arena);
+std::string configuration = R"EOF(
+  {
+    "NAME":"test_pod",
+    "NAMESPACE":"test_namespace",
+    "LABELS": {
+        "app": "productpage",
+        "version": "v1",
+        "pod-template-hash": "84975bc778"
+    },
+    "OWNER":"test_owner",
+    "WORKLOAD_NAME":"test_workload",
+    "PLATFORM_METADATA":{
+        "gcp_project":"test_project",
+        "gcp_cluster_location":"test_location",
+        "gcp_cluster_name":"test_cluster"
+    },
+    "ISTIO_VERSION":"istio-1.4",
+    "MESH_ID":"test-mesh"
+  }
+  )EOF";
+
+const static char encodeLookup[] =
+    "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/";
+const static char padCharacter = '=';
+
+std::string base64Encode(const uint8_t* start, const uint8_t* end) {
```

Comment:
There is `Base64::encode()` already available in Envoy, please use that instead of re-implementing it.

Reply:
I think the issue is that it is difficult to create a Wasm build if using a dependency in Envoy.

Reply:
Oops, sorry, I thought it was in the speed test executor, not inside Wasm module. Yeah, that's fine then. Sorry for the noise!